src/main/scala/firrtl/transforms/InlineBitExtractions.scala

```diff
@@ -35,10 +37,10 @@ object InlineBitExtractionsTransform {
     case _ => expr // Not a candidate
   }
 
-  /** Mapping from references to the [[firrtl.ir.Expression Expression]]s that drive them */
+  /** Mapping from references to the [[firrtl.ir.Expression]]s that drive them */
```

Comment:
I actually prefer this one how it was, that's intentional syntax so that the text shows up as `Expression` with the correct link rather than as `firrtl.ir.Expression`, same with the ones below (a couple of which may have been broken because they went over multiple lines.

Reply:
I'll go back through and check, I think at one point there seemed to be situation where that form was causing a warning, but it could have been a mixed up with some other idiosyncrasy of scaladoc

Reply:
That seems ok, it does not work however in situations where this works
```
    * @throws scala.IllegalArgumentException if vprime is not a subset of V
```
but this gives a warning
```
    * @throws [[scala.IllegalArgumentException IllegalArgumentException]] if vprime is not a subset of V
```

Reply:
I updated all the files I changed to use your preferred form

Reply:
The `@throws ExceptionName` is the correct, undocumented syntax. The first argument is special and isn't supposed to be a link.

Unrelated, I usually write this as the underlying Java exception as opposed to the Scala type alias. That doesn't matter, though.